simple-ipc.c

```diff
@@ -182,4 +182,143 @@ int ipc_send_command(const char *path, const char *message, struct strbuf *answe
 		close(fd);
 	return ret < 0 ? -1 : 0;
 }
+#elif !defined(NO_UNIX_SOCKETS)
+#include "unix-socket.h"
+#include "sigchain.h"
+
+static const char *fsmonitor_listener_path;
```

Comment:
Do we want this path to be named specifically with `fsmonitor`, or do we want something more generic?

Reply:
More generic!